### ISSUE TITLE
Emoji modifiers don't seem to work.

### ISSUE THREAD

**Issue by hex2f** [omnidan/node-emoji#59]
# Problem
I want to unemojify emojis that have modifiers (gender, color) but for some reason it doesn't even detect these as emojis.
# Example
`require('node-emoji').hasEmoji('🤦‍♂')   =>   false`
`require('node-emoji').find('🤦‍♂')   =>   undefined`
# Expected
`require('node-emoji').hasEmoji('🤦‍♂')   =>   true`
`require('node-emoji').find('🤦‍♂')   =>   ":man_facepalming:"`

**Comment by omnidan** [comment]
this is not implemented right now, also see #57. Unfortunately I don't have much time myself right now, but PRs are welcome 😁

**Comment by timkinnane** [comment]
Hey @omnidan - this is a cool lib and I'd like to contribute. Would you mind updating the readme to separate the implemented features from those still on the roadmap as a starting point.

I only checked the issues after some time debugging my usage to find why they were returning undefined.

**Comment by JoshuaKGoldberg** [comment]
I just tried this out with `node-emoji@v2.0.2` and it works! 🙌 

```js
import * as emoji from "node-emoji";

console.log(emoji.has("🤦‍♂"));
console.log(emoji.find("🤦‍♂"));
```

Must have been resolved by #113 switching to `emojilib` for emojis. Yay!
